sql_generators/active_users/templates/desktop_checks.sql

```diff
@@ -94,8 +94,18 @@ live_table_qdau_count AS (
 )
 SELECT
   IF(
-    (SELECT * FROM live_table_qdau_count) <> (SELECT * FROM qdau_sum),
-    ERROR("QDAU mismatch between aggregates table and live table"),
+    ABS((SELECT * FROM qdau_sum) - (SELECT * FROM live_table_qdau_count)) > 10,
+    ERROR(
+      CONCAT(
+        "QDAU mismatch between the live (`telemetry_live.main_v5`) and active_users_aggregates (`{{ dataset_id }}.{{ table_name }}`) tables is greated than 10." ,
+        " Live table count: ",
+        (SELECT * FROM live_table_qdau_count),
+        " | active_users_aggregates (QDAU): ",
+        (SELECT * FROM qdau_sum),
+        " | Delta detected: ",
+        ABS((SELECT * FROM qdau_sum) - (SELECT * FROM live_table_qdau_count))
```

Comment:
The delta is useful, thanks for adding it. So are the totals, I woudn't remove them

Reply:
The totals are still there. I made a small tweak and added all channel tables to the message.